play-scala-slick-example/src/main/scala/SlickExamplePlaceholder.scala

```diff
@@ -0,0 +1,4 @@
+
+object SlickExamplePlaceholder extends App {
+
+}
```

Comment:
@Max-AR What is this file about?

Reply:
I couldn't get the project to compile since I mis-configured `build.sbt` in the root so I added that file. But with your new config it will compile fine without this file here. Thanks for updating that, it's much cleaner now.